README.md

```diff
@@ -1,42 +1,50 @@
+
+These values are consumed by dashboards such as the one hosted at [telemetry.mozilla.org](http://telemetry.mozilla.org).
 
 Consuming Telemetry Aggregations
 --------------------------------
-Include into your code `http://telemetry.mozilla.org/v1/telemetry.js` feel free
-to use the other modules too.
-Don't go about reading the raw JSON files, they are not designed for human
-consumption!
-
-
-Hacking Telemetry Dashboard
----------------------------
-If you want to improve the user-interface for telemetry dashboard, clone this
-repository, setup a static server that hosts the `html/` folder on our localhost
-and start hacking. This is easy!
-
-If you want to add new aggregations, or improve upon existing aggregations,
-change the storage format, take a look at `Formats.mkd`. Talk to the guy who is
-maintaining telemetry dashboard.
-
-Basic flow is as follows:
-  1. An `.egg` file is generated with `make egg`
-  2. Analysis tasks are created with telemetry-server
-  3. `DashboardProcessor` from `analysis.py` aggregated telemetry submissions,
-     this process is driven by telemetry-server.
-  4. `Aggregator` from `aggregator.py` collects results from analysis tasks, by:
-    1. Downloads existing data from s3
-    2. Fetch task finished messages from SQS
-    3. Download `result.txt` files in parallel
-    4. Updates results on disk
-    5. Publishes updated results in a new subfolder of `current/` on s3, every
-       once in a while.
-    6. Check points all aggregated data to a subfolder of `check-points/` on s3,
-       every once in a while.
-    7. Repeat
+The aggregated data is available for third-party applications via the `telemetry.js` library - it is not necessary to run this analysis job yourself.
+
+For consuming the results of the aggregations, see the [telemetry.js documentation](https://telemetry.mozilla.org/docs.html#Telemetry).
+
+Avoid using or referencing the JSON data directly, as these may be changed or removed without warning.
+
+Deploying Telemetry Aggregator/Telemetry Server
+-----------------------------------------------
+
+Telemetry Aggregator/Server is written for deployment on AWS.
+
+* Provision an on-demand AWS machine (this document will assume a Ubuntu/Debian-derived medium instance).
+* Copy the contents of `scripts/` onto the server into `~/`.
+* See `~/README.md` for further setup instructions.
+
+Hacking Telemetry Aggregator
+----------------------------
+To improve the user-interface for telemetry dashboard, see the README for the [Telemetry Dashboard repository](https://github.com/mozilla/telemetry-dashboard).
+
+To add new aggregations, improve upon existing aggregations, or change the storage format, take a look at `Formats.mkd`. Additionally, make sure to talk to the telemetry dashboard maintainer(s).
+
+Setting up the development environment:
+
+  1. Install Subversion and LibLZMA.
```

Comment:
Oh, that's true...

@Uberi, maybe we should move to use https://github.com/miloyip/rapidjson for rapidjson, google code is shutting down....